File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/context_create_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ContextCreateRequest:
    """Context creation request"""
    model: str
    messages: List[ContextMessage]
    mode: str = None
    ttl: int = None
    truncation_strategy: TruncationStrategy = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ContextCreateRequest.kt
package com.sdkwork.ai

data class ContextCreateRequest(
    val model: String? = null,
    val messages: List<ContextMessage>? = null,
    val mode: String? = null,
    val ttl: Int? = null,
    val truncationStrategy: TruncationStrategy? = null
)
